src/components/styles/careersection.css

```diff
@@ -0,0 +1,21 @@
+
+.tableC{
+    margin:15%;

```

Comment:
I don't recommend to use percentage for margins or paddings. Instead you can use rem, em, and even px with breakpoints. Like px-4 sm:px-16 lg:px-[10.25rem]. 
However both career sections should have same indentation and spacing, so lets wait to merge both of them and see how we can adjust them to have same values

Reply:
alright so need for me now to edit this, right? @MhamadR

Reply:
We can keep this margin for now and change it later when we merge both your component and Zoriana's component.
This way we can make sure that both of these components are styled the same.